app/server/appsmith-server/src/main/java/com/appsmith/server/services/ce/TenantServiceCEImpl.java

```diff
@@ -27,15 +29,18 @@ public class TenantServiceCEImpl extends BaseService<TenantRepository, Tenant, S
 
     private final ConfigService configService;
 
+    private final SessionUserService sessionUserService;
+
     public TenantServiceCEImpl(Scheduler scheduler,
                                Validator validator,
                                MongoConverter mongoConverter,
                                ReactiveMongoTemplate reactiveMongoTemplate,
                                TenantRepository repository,
                                AnalyticsService analyticsService,
-                               ConfigService configService) {
+                               ConfigService configService, SessionUserService sessionUserService) {
```

Comment:
Can we keep consistent formatting please?

```suggestion
                               ConfigService configService,
                               SessionUserService sessionUserService) {
```